hydra-plutus/src/Hydra/Contract/Initial.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -fno-specialize #-}
+
+-- | Contract for Hydra controlling the redemption of participation tokens.
+module Hydra.Contract.Initial where
```

Comment:
Just a dummy module to use in the `ContractSM`. Ideally, we want to make it look a bit more like the initial validator in the OnChain module.